CRM/Dedupe/BAO/DedupeRuleGroup.php

```diff
@@ -399,10 +399,10 @@ private static function getRuleFieldFilter(array $rule): string {
    * @param array $contactIDs
    * @param array $params
    *
-   * @return void
+   * @return bool
    * @throws \Civi\Core\Exception\DBQueryException
    */
-  public function fillTable(int $id, array $contactIDs, array $params): void {
+  public function fillTable(int $id, array $contactIDs, array $params): bool {
```

Comment:
so looking at L521 we might be returning void here?

Reply:
actually maybe not i'm not sure

Reply:
no - it still should get to the TRUE on line 524